app/controllers/kinde_sdk/auth_controller.rb

```
@@ -137,16 +207,34 @@ def validate_nonce(id_token, original_nonce, issuer, client_id)
-      false
+      nil
+    end
+
+    # Fetches the JSON Web Key Set (JWKS) from the issuer
+    # @param issuer [String] The token issuer
+    # @return [Hash] The JWKS data or nil if fetch fails
+    def fetch_jwks(issuer)
+      jwks_uri = URI.parse("#{issuer}/.well-known/jwks.json")
+      jwks_response = Net::HTTP.get(jwks_uri)
+      JSON.parse(jwks_response)
+    rescue StandardError => e
```

RuboCop (1.73)

[warning] Useless assignment to variable - `e`.

(Lint/UselessAssignment)